yjit_asm.h

```diff
@@ -242,8 +256,8 @@ x86opnd_t const_ptr_opnd(const void *ptr);
 
 // Machine code allocation
 uint8_t* alloc_exec_mem(uint32_t mem_size);
-uint8_t* alloc_code_page();
-void free_code_page(uint8_t* page_ptr);
+code_page_t* alloc_code_page();
```

Comment:
You probably want `(void)` here. Reasoning https://gist.github.com/XrXr/ad1dfe0cfb91dfc167672fef1c5f3157